### ISSUE TITLE
Fix broken builds on both master and development branches

### ISSUE THREAD

**Issue by avishnyakov** [PowerShell/PowerShellGetv2#585]
Builds been failing for 13 days and 2 months for master/development branch.
* https://github.com/PowerShell/PowerShell

Examples:
* https://ci.appveyor.com/project/PowerShell/powershellget/branch/master
* https://ci.appveyor.com/project/PowerShell/powershellget-0lib3/branch/development

![image](https://user-images.githubusercontent.com/3785585/74478775-1c353780-4f02-11ea-89ca-759fd49d0c25.png)

**Comment by SydneyhSmith** [comment]
Thanks @avishnyakov this has been on our backlog as our focus has been on PowerShellGet 3.0, but we are working to resolve this error.

**Comment by avishnyakov** [comment]
Right, noted. Was not this scrum/agile thing all about drop everything and fix the broken build? 😕

**Comment by alerickson** [comment]
@avishnyakov the build itself is not broken, they're flakey pester tests that need to be updated.  But you're right, this should be addressed and we'll get to this in the immediate future.  Unfortunately the CI testing for PowerShellGet needs a bit of work, but we'll need to make some changes, particulary on AppVeryor, soon so we'll certainly address this issue then.

**Comment by avishnyakov** [comment]
All good. Thank you for sharing this, @alerickson.

**Comment by avishnyakov** [comment]
@alerickson , could you please also comment on https://github.com/PowerShell/PowerShellGet/issues/584 
"Yep, saw this" would be more than enough so we can propagate "Microsoft saw this" back to the team?
